build.sbt

```diff
@@ -1,17 +1,18 @@
 ThisBuild / tlBaseVersion := "0.23"
-ThisBuild / tlMimaPreviousVersions ++= (0 to 11).map(y => s"0.23.$y").toSet
+ThisBuild / tlMimaPreviousVersions ++= (0 to 23).map(y => s"0.23.$y").toSet
```

Comment:
This should stay fixed at 11. From 0.23.12+ we no longer published http4s-play-json from the http4s/http4s repo.

Reply:
Ah yes I see, correct I believe? Thanks